wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -4,6 +4,11 @@
       "id": "propertyList",
       "type": "object",
       "javaType": "java.util.Map<String, String>"
+    },
+    "selector" : {
+      "id": "selector",
+      "type": "object",
```

Comment:
Do we want to allow a free-form object or rather something along the lines of:

```json
"type": "object",
"additionalProperties": {
  "type": "string"
}
```

i.e. an object with any keys that are only allowed to have string values?

Reply:
Well, that's interesting way to limit values type. Makes sense to add it. Tks.